### PR TITLE
Add optional anchor point to resize

### PR DESCRIPTION
**Pull request by sWW26**
Added ability to manually define the specific anchor point when resizing in `BoxPad` mode

### CODE REVIEW

File: src/ImageProcessor/Imaging/Resizer.cs [hunk 1]
```diff
@@ -234,44 +238,60 @@ protected virtual Bitmap ResizeLinear(Image source, int width, int height, Recta
...
+                                case AnchorPosition.Bottom:
+                                    destinationY = height - sourceHeight;
+                                    destinationX = (width - sourceWidth) / 2;
+                                    break;
+                                case AnchorPosition.BottomLeft:
+                                    destinationY = height - sourceHeight;
+                                    destinationX = 0;
+                                    break;
+                                default:
+                                    if (centerCoordinates != null && centerCoordinates.Any())
```
Comment (dlemstra) [inline src/ImageProcessor/Imaging/Resizer.cs]: Can you clarify why you also added this?

Comment (sWW26) [inline src/ImageProcessor/Imaging/Resizer.cs]: It's been a long day.  Should have been removed.

File: src/ImageProcessor/Imaging/Resizer.cs [hunk 2]
```diff
@@ -234,44 +239,52 @@ protected virtual Bitmap ResizeLinear(Image source, int width, int height, Recta
...
-                                break;
-                            case AnchorPosition.BottomLeft:
-                                destinationY = height - sourceHeight;
-                                destinationX = 0;
-                                break;
-                            default:
-                                destinationY = (height - sourceHeight) / 2;
-                                destinationX = (width - sourceWidth) / 2;
-                                break;
+                            destinationY = anchorPoint.Value.Y;
```
Comment (JimBobSquarePants) [inline src/ImageProcessor/Imaging/Resizer.cs]: There's no sanitation here for the X and Y values. We'll need that before we can continue.

**Comment by JimBobSquarePants** [comment]
Hi @sWW26 

Thanks for this. What's the use case? We normally have a bit of a chat on Gitter before adding features.

**Comment by sWW26** [comment]
Hi,

This might not be the best way of doing this but I couldn't seem to find another way.  I needed a way to increase the size of an image by creating the new increased size container then placing the image inside it.

For example taking this image

hn

and placing it inside a larger box in a specific position

hn-new

**Comment by JimBobSquarePants** [comment]
@sWW26 Doesn't look like you need to rebase anything here so will do a final check and merge over the weekend. Thanks! 💯

**Pull request merged**